src/frontend/src/common/analytics/eventTypes.ts

```diff
@@ -82,13 +85,19 @@ export enum EVENT_TYPES {
  * that kicks off the underlying Segment `track` by ensuring that all events
  * extend from that base structure.
  */
-type EventValue = string | number | boolean | null;
+type EventValue = string | number | boolean | FilterDate | null;
```

Comment:
This should not be expanded. Because of how the data lands from Segment -> Snowflake, these are really the only kind of values that should ever be allowed. Adding a new kind of type here causes a lot of downstream pain, so it really should ONLY be string | number | boolean | null

Reply:
ok removed